runtime.txt

```diff
@@ -0,0 +1 @@
+python-2.7.14
```

Comment:
@phillipahereza Do you think 3.6 might offer performance improvements?

Reply:
@DavidLemayian but HURUmap runs on 2.7.14, wouldn't it be a safe bet to use the same version?

Reply:
@phillipahereza Safe bets are the enemy of progress. ;)

Reply:
@DavidLemayian in that case, should we also checkout django 2.0?